server/s3_put_chunk_upload_object_action.cc

```diff
@@ -316,6 +316,14 @@ void S3PutChunkUploadObjectAction::validate_put_chunk_request() {
   s3_log(S3_LOG_DEBUG, nullptr, "Exiting\n");
 }
 
+void S3PutChunkUploadObjectAction::_set_layout_id(int layout_id) {
+  assert(layout_id > 0 && layout_id < 15);
```

Comment:
layout id max is 9 right ?

Reply:
14 in s3config

Reply:
ok
In s3_obj_layout_mapping.yaml we have max till 9. I feel currently max is 9 and can go up later.